### DISCUSSION TITLE
Copying a folder server

### DISCUSSION THREAD

**Discussion by psynezo** [itzg/docker-minecraft-server#1684] [Q&A] [answered]
Hello, how to start a docker container and copy the contents of the server folder but not use the given folder?
I need to start a new container using the data from the server folder, but start the container as a separate server without interacting with the already existing folder.

**Comment by itzg** [comment, marked as answer]
Did you look at the documentation, such as

https://github.com/itzg/docker-minecraft-server#cloning-world-from-a-container-path

https://github.com/itzg/docker-minecraft-server#optional-plugins-mods-and-config-attach-points

**Reply by psynezo** [reply]
Thanks for the advice, I'll try this.

**Reply by psynezo** [reply]
I specified the plugins path: -e plugins={path} but it didn't work. For you to understand, every time I need to create a new container, that is, every time a new / data and I need to clone at least plugins into a new container, if we could clone all the files in the folder and specify new parameters, for example -e SERVER_PORT={port}, then that would be even better. I hope for your help.

**Reply by psynezo** [reply]
To be even more precise, I need to take the data from the named volume and clone it into an anonymous one, and if possible specify the port and other data.

**Reply by itzg** [reply]
That's what those sections describe. The source of the mounted container directories can be whatever you want, including named volumes.
